src/Microsoft.NET.Sdk.Aspire/WorkloadManifest.in.targets

```diff
@@ -5,4 +5,14 @@
     Condition=" ('$(IsAspireHost)' == 'true') and ('$(SkipAspireWorkloadManifest)' != 'true') "
   />
 
+  <Import
+    Project="Sdk.targets" Sdk="Aspire.Hosting.Orchestration"
+    Condition=" ('$(IsAspireHost)' == 'true') and ('$(SkipAspireWorkloadManifest)' != 'true') "
+  />
```

Comment:
We never got around to moving the Aspire.Hosting.Orchestration pack SDK import into the manifest targets; this seemed like a good time to finally update.